ratis-grpc/src/main/java/org/apache/ratis/grpc/server/GrpcLogAppender.java

```diff
@@ -407,7 +407,9 @@ public void onNext(InstallSnapshotReplyProto reply) {
           LOG.error("{}: Configuration Mismatch ({}): Leader {} has it set to {} but follower {} has it set to {}",
               this, RaftServerConfigKeys.Log.Appender.INSTALL_SNAPSHOT_ENABLED_KEY,
               getServer().getId(), installSnapshotEnabled, getFollowerId(), !installSnapshotEnabled);
+          break;
         case UNRECOGNIZED:
+          break;
```

Comment:
Should there be a LOG printed when hitting `UNRECOGNIZED`?

Reply:
@amaliujia Yeah, thanks you the suggestion, done.